demisto_sdk/commands/setup_env/setup_environment.py

```diff
@@ -92,27 +94,180 @@ def get_docker_python_path(docker_prefix: str) -> List[str]:
     return docker_python_path
 
 
-def update_dotenv(values: Dict[str, str], quote_mode="never"):
-    env_vars = dotenv.dotenv_values(DOTENV_PATH)
+def update_dotenv(file_path: Path, values: Dict[str, str], quote_mode="never"):
+    """
+    Configure the .env file with the given values. Generates the file if it doesn't exist.
+
+    Args:
+        file_path (Path): The path to the .env file
+        values (Dict[str, str]): The values to set
+        quote_mode (str): The quote mode to use. Defaults to "never"
+    """
+    file_path.touch(exist_ok=True)
```

Comment:
Doesn't the dotenv library create the dotenv for you?

Reply:
I just kept the existing code from [here](https://github.com/demisto/demisto-sdk/pull/3917/files#diff-149df266deb904266196e8819d10b53d761b3fb2a4f54ed4003e54727f04adf3L110) which creates the file.

I think it's better to keep it to be more error-resilient. If the file already exists, nothing will happen. If it doesn't, for some reason, it will avoid an unexpected error.